### PR TITLE
About view

### PR DESCRIPTION
**Pull request by alexvargas01**
about view has a custom component (CardGrid) that displays the team information and some placeholder text

### CODE REVIEW

File: src/views/About.jsx [hunk 1]
```diff
@@ -1,6 +1,23 @@
-import { Typography } from '@mui/material';
 import React from 'react';
+import Typography from '@mui/material/Typography';
 
-const AboutView = () => <Typography variant="h3">About</Typography>;
+import CardGrid from '../components/CardGrid';
+
+const AboutView = () => (
+  <div>
```
Comment (PatoGuereque) [inline src/views/About.jsx]: Use react fragment instead of div
```
<>
  // child nodes
</>
```

File: src/components/CardGrid.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      display="flex"
+      direction="row"
+      alignItems="center"
+      justifyContent="center"
+      rowSpacing={2}
+      columnSpacing={{ xs: 1, sm: 2, md: 3 }}
+      padding="20px"
+      margin="5px"
+    >
+      <Grid item md={3}>
```
Comment (PatoGuereque) [inline src/components/CardGrid.jsx]: Missing breakpoint for xs screens (mobile).

Should we have one card per row on mobile or two?

(``xs={6}`` or ``xs={12}``)

File: src/components/CardGrid.jsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            </Typography>
+            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+              adjective
+            </Typography>
+            <Typography variant="body2">
+              well meaning and kindly.
+              <br />a benevolent smile
+            </Typography>
+          </CardContent>
+        </Card>{' '}
```
Comment (PatoGuereque) [inline src/components/CardGrid.jsx]: Any reason for the ``{' '}``?

**Review by PatoGuereque — CHANGES_REQUESTED** [review]

File: src/components/CardGrid.jsx (hunk 2 above)
Comment (alexvargas01) [reply src/components/CardGrid.jsx]: one card per row on mobile

**Review by PatoGuereque — APPROVED** [review]
LGTM!

**Pull request merged**